frontend/packages/eslint-plugin-console/lib/config/prettier.js

```diff
@@ -0,0 +1,49 @@
+const merge = require('merge');
+
+module.exports = {
+  extends: ['prettier', 'prettier/react'],
+
+  plugins: ['prettier'],
+
+  rules: {
+    'prettier/prettier': [
+      'error',
+      {
+        singleQuote: true,
+        trailingComma: 'all',
+        bracketSpacing: true,
+        jsxBracketSameLine: false,
+        arrowParens: 'always',
+        printWidth: 120,
+        semi: true,
+        useTabs: false,
+      },
+      {
+        usePrettierrc: false,
+      },
+    ],
+  },
+
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: merge(require('./rules/prettier-typescript'), {
+        'prettier/prettier': [
+          'error',
+          {
+            parser: 'typescript',
+            singleQuote: true,
+            trailingComma: 'all',
+            bracketSpacing: true,
+            jsxBracketSameLine: false,
+            arrowParens: 'always',
+            printWidth: 100,
```

Comment:
Is there a reason for making the width incnsistent between `.js` and `.ts` files?

Reply:
No! they should be the same. Forgot to update the typescript one ;/

Reply:
https://github.com/openshift/console/pull/1659